src/assets/en.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Should these files be under a `i18n` folder?

Or as an alternative the path should be changed accordingly on the translator module
```
return new TranslateHttpLoader(http, "app/rapydo/assets/i18n/", ".json");
```

Reply:
yes sure! I forgot to create that i18n folder